ui/raidboss/data/04-sb/trial/tsukuyomi-ex.ts

```diff
@@ -370,35 +369,40 @@ const triggerSet: TriggerSet<Data> = {
         'Specter of Zenos': 'spectre de Zenos',
       },
       'replaceText': {
+        '\\?': ' ?',
+        '\\(E->W\\)': '(E->O)',
+        '\\(SW->NW\\)': '(SO->NO)',
+        '\\(gun\\)': '(fusil)',
+        '\\(gun/spear\\)': '(fusil/lance)',
         '\\(spear\\)': '(lance)',
         'Antitwilight': 'Belle-de-nuit',
-        'Bright Blade': 'Lame blafarde',
+        'Bright Blade/Dark Blade': 'Lame blafarde/ténébreuse',
         'Concentrativity': 'Kenki concentré',
         'Crater': 'Explosion de fragment lunaire',
         'Dance Of The Dead': 'Danse des morts',
-        'Dark Blade': 'Lame ténébreuse',
         'Dispersivity': 'Onde Kenki',
         'Empire adds': 'Adds Impériaux',
         'Hagetsu': 'Pulvérisation lunaire',
         'Homeland adds': 'Adds Domiens',
-        'Lead Of The Underworld': 'Tir de l\'au-delà',
+        'Lead Of The Underworld(?!/Steel Of The Underworld)': 'Tir de l\'au-delà',
+        'Lead Of The Underworld/Steel Of The Underworld': 'Tir/Pointes de l\'au-delà',
         'Lunacy': 'Efflorescence au clair de lune',
         'Lunar Halo': 'Flamboiement lunaire',
         'Lunar Rays': 'Rayons lunaires',
         'Midnight Rain': 'Bruine nocturne',
         'Moonbeam': 'Faisceau lunaire',
         'Moonfall': 'Impact de fragment lunaire',
         'Nightbloom': 'Lis araignée',
+        'Nightfall': 'Jeune nuit',
         'Perilune': 'Zénith lunaire',
         'Reprimand': 'Correction',
-        'Steel Of The Underworld': 'Pointes de l\'au-delà',
+        '(?<!Lead Of The Underworld/)Steel Of The Underworld': 'Pointes de l\'au-delà',
```

Comment:
Same.

Reply:
```suggestion
        '(?<!/)Steel Of The Underworld': 'Pointes de l\'au-delà',
```

Similarly, does this work?